webapp/portlet/src/main/webapp/space-menu/extension.js

```diff
@@ -2,4 +2,10 @@ function getExtensionsByType(type) {
   return extensionRegistry.loadExtensions('space-menu', type);
 }
 
-export const spaceMenuActionComponents = getExtensionsByType('action-component');
\ No newline at end of file
+export const spaceMenuActionComponents = getExtensionsByType('action-component');
+
+function getSpaceTitleExtensionsByType(type) {
+  return extensionRegistry.loadExtensions('space-title-action-components', type);
+}
+
+export const spaceTitleActionComponents = getExtensionsByTypeCallButton('action-component');
```

Comment:
you need to change the function name here aldo

Reply:
Already